usaspending_api/api_contracts/contracts/v2/references/filter_tree/tas.md

```diff
@@ -10,19 +10,23 @@ database fields based on depth in the tree.
 ## Toptier Search [GET /api/v2/references/filter_tree/tas/{?depth}]
 
 Returns a list of toptier agencies that have at least one TAS affiliated with them
-+ Request A request with a contract id
++ Request (application/json)
     + Parameters
-        + `depth`: `0` (optional, enum[number]) How many levels deep the search will populate each tree. 
+        + `depth` (optional, enum[number]) 
+        How many levels deep the search will populate each tree. 
             + Members
                     + `0`
                     + `1`
                     + `2`
         0 will return only agencies, 1 will return agencies and any federal accounts under them, and so on.
+        + `filter` (optional, string) 
+        When provided, only results who's id or name matches the provided string (case insensitive) will be returned, along with any ancestors to a matching node. 
```

Comment:
`who's` -> `whose`, or "only results with an id or name matching the provided string..." (similar for federal account)